Code/BuildSystem/CMake/Platforms/Detect_Linux.cmake

```diff
@@ -0,0 +1,44 @@
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+
+        set_property(GLOBAL PROPERTY EZ_CMAKE_PLATFORM_NAME "Linux")
+		set_property(GLOBAL PROPERTY EZ_CMAKE_PLATFORM_PREFIX "Linux")
+		set_property(GLOBAL PROPERTY EZ_CMAKE_PLATFORM_POSTFIX "Linux")
+
+endif()
+
+
+macro(ez_platform_set_build_flags_clang)
+	target_compile_options(${TARGET_NAME} PRIVATE -fPIC)
+
+	# Look for the super fast ld compatible linker called "mold". If present we want to use it.
+	find_program(MOLD_PATH "mold")
```

Comment:
the mold detection is duplicated between Configure_Linux and Detect_Linux, is this intnetinal? Could we deduplicate?

Reply:
That was an accident (unsaved file in VSCode)